Language/Variables/Evaluation_of_Implicit_Variable_Getters/definition_A01_t03.dart

```diff
@@ -0,0 +1,101 @@
+// Copyright (c) 2021, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// @assertion Case ⟨Static or library variable⟩. If `d` declares a static or
+/// library variable, the implicitly induced getter of `id` executes as follows:
+/// - Non-constant variable with an initializer. In the case where `d` has an
+///   initializing expression and is not constant, the implicitly induced getter
+///   of `id` is a late-initialized getter. This determines the semantics of an
+///   invocation.
+///
+/// @description Checks that it is not an error if during an evaluation of the
```

Comment:
Typo:
```suggestion
/// @description Checks that it is an error if during an evaluation of the
```

This is actually a case which is covered by dart-lang/sdk#50383, and the rule is even stronger than the description here: It is enough that, for example, the declaration of `sVar` has an initializing expression that contains a reference to `sVar`, the evaluation of the initializing expression doesn't have to need that value (which is in fact the case here, because `func` doesn't invoke its argument).

Reply:
So the error expectations below are fine, but the description could be strengthened to say that a circular dependency is an error even when it is only syntactic, and there's no actual infinite loop during evaluation.